File: front-end/src/app/service-models/app/v1/general/contact-us-s-m.ts
import { CoinManagementServiceModelBase } from '../../base/coin-management-service-model-base';

export class ContactUsSM extends CoinManagementServiceModelBase<number> {
    name!: string;
    subject!: string;
    message!: string;
    emailId!: string;
    phone!: string;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/General/ContactUsSM.cs
namespace Siffrum.Web.Payroll.ServiceModels.v1.General
{
    public class ContactUsSM : Base.SiffrumPayrollServiceModelBase<int>
    {
        public string Name { get; set; }
        public string Subject { get; set; }
        public string Message { get; set; }
        public string EmailId { get; set; }
        public string Phone { get; set; }
    }
}
